services/webhook/tests/test_review_latency.py

```
@@ -0,0 +1,64 @@
+
+
+def test_percentile_nearest_rank():
+    vals = [1.0, 2.0, 3.0, 4.0, 5.0]
+    assert percentile(vals, 50) == 3.0
+    assert percentile(vals, 95) == 5.0
+    assert percentile([], 50) is None
+
+
+def test_summarize_trials_groups_and_p50():
+    trials = [
+        TrialResult(1, "small", "cave", 0.1, 1.0, True, False, 100, 50),
```

Ruff (0.15.21)

[warning] Boolean positional value in function call

(FBT003)